user-interface/src/case-detail/CaseDetailScreen.test.tsx

```diff
@@ -666,4 +680,364 @@ describe('Case Detail screen tests', () => {
+      petitionLabel: 'Voluntary Petition',
+      closedDate: '01-08-1963',
+      dismissedDate: '01-08-1964',
+      assignments: [],
+      debtor: {
+        name: 'Roger Rabbit',
+      },
+      debtorAttorney: {
+        name: 'Jane Doe',
+        address1: '123 Rabbithole Lane',
+        cityStateZipCountry: 'Ciudad Obregón GR 25443, MX',
+        phone: '234-123-1234',
+      },
+    };
+
+    test('should fix when scrolled at top of viewport', async () => {
+      render(
+        <BrowserRouter>
+          <div className="App" data-testid="app-component-test-id">
+            <header
+              className="cams-header usa-header-usa-header--basic"
+              style={{ minHeight: '100px', height: '100px' }}
+              data-testid="cams-header-test-id"
+            ></header>
+            <CaseDetail caseDetail={testCaseDetail} />
+            <div style={{ minHeight: '2000px', height: '2000px' }}></div>
+          </div>
+        </BrowserRouter>,
+      );
+
+      const app = await screen.findByTestId('app-component-test-id');
+
+      await waitFor(() => {
+        const pane = document.querySelector('.left-navigation-pane-container');
+        expect(pane).toBeInTheDocument();
+        expect(pane).not.toHaveClass('fixed');
+      });
+
+      const paneBeforeBreak = document.querySelector('.left-navigation-pane-container');
+      expect(paneBeforeBreak).toBeInTheDocument();
+
+      window.HTMLElement.prototype.getBoundingClientRect = () =>
+        ({
+          top: 2,
+        }) as DOMRect;
+      fireEvent.scroll(app as Element, { target: { scrollTop: 98 } });
+
+      expect(paneBeforeBreak).toBeInTheDocument();
+      expect(paneBeforeBreak).not.toHaveClass('fixed');
+
+      window.HTMLElement.prototype.getBoundingClientRect = () =>
+        ({
+          top: -175,
+        }) as DOMRect;
+      fireEvent.scroll(app as Element, { target: { scrollTop: 275 } });
+
+      const paneAfterBreak = document.querySelector('.left-navigation-pane-container');
+      expect(paneAfterBreak).toBeInTheDocument();
+      expect(paneAfterBreak).toHaveClass('fixed');
+    });
+  });
+
+  describe('Docket entry sorter', () => {
+    const left: CaseDocketEntry = {
+      sequenceNumber: 0,
+      dateFiled: '',
+      summaryText: '',
+      fullText: '',
+    };
+    const right: CaseDocketEntry = {
+      sequenceNumber: 1,
+      dateFiled: '',
+      summaryText: '',
+      fullText: '',
+    };
+    test('should return the expected sort direction for Newest sort', () => {
+      const fn = docketSorterClosure('Newest');
+      const expectedValue = 1;
+      expect(fn(left, right)).toEqual(expectedValue);
+    });
+    test('should return the expected sort direction for Oldest sort', () => {
+      const fn = docketSorterClosure('Oldest');
+      const expectedValue = -1;
+      expect(fn(left, right)).toEqual(expectedValue);
+    });
+  });
+
+  describe('old tests we may have to rewrite', () => {
```

Comment:
Now that we're done with these tests, we should rename the describe. 
```suggestion
  describe('sort, search, and filter tests', () => {
```